app/client/cypress/e2e/Regression/ClientSide/ActionExecution/ActionSelector_DownloadFunctions_spec.ts

```
@@ -0,0 +1,271 @@
+      deployMode.DeployApp();
+      agHelper.AssertElementVisibility(appSettings.locators._header);
+      agHelper.ClickButton("Submit");
+      agHelper.ValidateToastMessage("Download Failed");
+      deployMode.NavigateBacktoEditor();
+      EditorNavigation.SelectEntityByName("Button1", EntityType.Widget);
+      propPane.ToggleJSMode("onClick", false);
+      agHelper.GetNClick(propPane._actionCardByTitle("Execute a JS function"));
+      agHelper.GetNClick(propPane._actionSelectorDelete);
+    });
+
+    it.only("3. To verify the download() function when invalid or null data is passed. The download should fail, or an appropriate error message should be logged.", () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)